src/css/Card.css

```diff
@@ -1,3 +1,51 @@
+/* Testing Purpose */
 .Card {
-  background-color: skyblue;
+  /* border: yellow solid 5px; */
+}
+/* ==============  */
+.Card {
```

Comment:
The cards look great! I would like to incorporate some of the pink from the background in the cards at some point too.